Userland/Utilities/dd.cpp

```diff
@@ -36,6 +40,29 @@ enum Status {
     Noxfer
 };
 
+Status status = Default;
+size_t total_bytes_copied = 0;
+size_t total_blocks_in = 0, partial_blocks_in = 0;
+size_t total_blocks_out = 0, partial_blocks_out = 0;
```

Comment:
Minor nit: Couldn't we create a struct that has all these parameters as members and pass it to closing statistics instead of creating global variables?

Reply:
Yeah, that is an alternative as well. I don't know how globals are treated in Userland code. So I am fine with what the maintainer wants :)